workflow-runtime/src/main/java/com/squareup/workflow1/WorkflowInterceptor.kt

```diff
@@ -81,10 +85,9 @@ public interface WorkflowInterceptor {
   public fun <P, S, O, R> onRender(
     renderProps: P,
     renderState: S,
-    context: BaseRenderContext<P, S, O>,
```

Comment:
I wonder if this removal was too aggressive. When the interceptor gets the actual context here, it can render its own workflow children or run side effects itself. Now it can't. Should we add it back in? I don't think it can hurt, and it's actually a less drastic change.

Reply:
Would we pass it back through `proceed()` as well? That seems redundant with the interceptor.

Reply:
Nope. The utility of passing the context here is simply to allow the interceptor to do workflow-y things with it.